common/djangoapps/enrollment/views.py

```diff
@@ -264,9 +267,12 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
 
                If honor mode is not supported for the course, the request fails and returns the available modes.
 
-               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified"
-               or "professional". If the mode is not supported for the course, the request will fail and return the
-               available modes.
+               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified",
+               "professional" or "credit". If the mode is not supported for the course, the request will fail and
+               return the available modes.
+
+               The enrollment attributes are also set against the course mode if provided 'enrollment_attributes' are
```

Comment:
@wedaly   ok thanks.  Something like:  "You can include other key:value parameters as enrollment attributes for specific course modes as needed. For example, for credit mode, you could include the parameter ``provider:UniversityX`` to specify the credit provider attribute.